Apps/WebClient/src/ClientApp/src/components/modal/AppTourComponent.vue

```diff
@@ -0,0 +1,202 @@
+        // Prevent modal from closing
+        bvModalEvt.preventDefault();
+        if (this.slideIndex < this.slides.length - 1) {
+            this.tourCarousel.next();
+        }
+    }
+
+    private previous(bvModalEvt: Event): void {
+        // Prevent modal from closing
+        bvModalEvt.preventDefault();
+        if (this.slideIndex > 0) {
+            this.tourCarousel.prev();
+        }
+    }
+
+    private get slides(): TourSlide[] {
+        return this.isMobile ? this.mobileSlides : this.desktopSlides;
+    }
+
+    private get currentSlide(): TourSlide | undefined {
+        return this.slides.length > 0
+            ? this.slides[this.slideIndex]
+            : undefined;
+    }
+
+    private get isFinalSlide(): boolean {
+        return this.slideIndex === this.slides.length - 1;
+    }
+
+    private get isFirstSlide(): boolean {
+        return this.slideIndex === 0;
+    }
+}
+</script>
+
+<template>
+    <b-modal
+        id="app-tour"
+        :visible="isVisible"
+        hide-footer
+        hide-header
+        centered
+        no-close-on-backdrop
+        body-class="p-0"
+        content-class="overflow-hidden"
+        size="lg"
+    >
+        <b-carousel
+            id="carousel-1"
+            ref="tourCarousel"
+            v-model="slideIndex"
+            :interval="0"
+            indicators
+        >
+            <b-carousel-slide
+                v-for="slide in slides"
+                :key="slide.title"
+                :img-src="slide.imageUri"
+                :img-alt="slide.imageAlt ?? slide.title"
+            />
+        </b-carousel>
+        <div class="p-3">
+            <b-row v-if="currentSlide">
+                <b-col>
+                    <h3>{{ currentSlide.title }}</h3>
+                    <p v-if="currentSlide.description">
+                        {{ currentSlide.description }}
+                    </p>
+                </b-col>
+            </b-row>
+            <b-row v-if="!isFinalSlide">
+                <b-col>
+                    <hg-button variant="link" @click="hideModal"
+                        >skip</hg-button
+                    >
+                </b-col>
+                <b-col class="d-flex justify-content-end">
+                    <hg-button
+                        :disabled="isFirstSlide"
+                        variant="secondary"
+                        @click="previous($event)"
+                        >back</hg-button
+                    >
+                    <hg-button
+                        variant="primary"
+                        class="ml-3"
+                        @click="next($event)"
+                        >Next</hg-button
+                    >
+                </b-col>
+            </b-row>
+            <b-row v-else>
+                <b-col class="d-flex justify-content-center">
+                    <hg-button variant="primary" class="ml-3" @click="hideModal"
+                        >Done</hg-button
+                    >
```

Comment:
Nit: code looks nicer with child content on separate line (and the extra whitespace added shouldn't make an impact here).
```suggestion
                    <hg-button variant="primary" class="ml-3" @click="hideModal">
                        Done
                    </hg-button>
```